packages/local_auth/ios/Classes/FLTLocalAuthPlugin.m

```diff
@@ -122,7 +122,10 @@ - (void)authenticateWithBiometrics:(NSDictionary *)arguments
   NSError *authError = nil;
   self.lastCallArgs = nil;
   self.lastResult = nil;
-  context.localizedFallbackTitle = @"";
+  context.localizedFallbackTitle = nil;
+  if (arguments[@"localizedFallbackTitle"] != (NSString *)[NSNull null]) {
```

Comment:
Why are you casting an `NSNull` to an `NSString`? They are completely different types.

Also, you should ideally handle both `nil` and `NSNull` here for safety.